setup.py

```diff
@@ -510,7 +510,7 @@ def resolveVars(filename, confvalues, force):
     name=CONFIG.NAME,
     version='{}.{}.{}'.format(MAJOR_VERSION, MINOR_VERSION, CONFIG.REVISION),
     description=CONFIG.DESCRIPTION,
-    long_description=CONFIG.DESCRIPTION,
+    long_description=open("README.md").read(),
```

Comment:
I would prefer something more like this:

```
try:
   long_desc =  open('README.md').read()
except:
  long_desc = CONFIG.DESCRIPTION

....
setup(
...
long_description=long_desc,
...
)

But maybe I am too cautious.

Reply:
Another idea would be to use a setup.cfg:

https://github.com/pandas-dev/pandas/blob/f7771fa079c3a0fbd1fb0ea84de4f3f8f1b7e314/setup.cfg

I‘ve never used that before, but would like to read into it. What do you think?

I think your description would lead to silent errors. Thatd be great for someone installing it on their machine, but in CI it would just pass silently and PyPI has the wrong description.

Reply:
Ok fair point. Concerning setup.py it is yet an other python  setup file and I prefer pyproject.toml